priv/repo/migrations/20160322222933_addnullchatprovider.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Cog.Repo.Migrations.Addnullchatprovider do
```

Comment:
@lhaskins Did you by chance run `mix ecto.gen.migration addnullchatprovider` instead of `mix ecto.gen.migration add_null_chat_provider`?